docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml

```diff
@@ -35,9 +35,9 @@ content:
     weight: 6
     label: above
     settings:
-      mirador_settings: ''
+      link_to_entity: false
     third_party_settings: {  }
-    type: mirador_text
```

Comment:
Have added Mirador field formatter in the new Display mode.

For default, we need the value of the Image field to generate the thumbnail, so we kept the field as plain text and can replace the image name with Thumbnail.